charts/redhat/redhat/redhat-developer-hub-orchestrator-infra/OWNERS

```diff
@@ -0,0 +1,14 @@
+chart:
+  name: redhat-developer-hub-orchestrator-infra
+  shortDescription: A Helm chart for deploying Red Hat Developer Hub Orchestrator Infra
+publicPgpKey: null
+users:
+  - githubUsername: rhdh-bot
+  - githubUsername: nickboldt
+  - githubUsername: schultzp2020
```

Comment:
should we add anyone from the orchestrator team instead of Paul and Tomas?

Reply:
I think it is fine not to add the Orchestrator team here. See my comment here: https://redhat-internal.slack.com/archives/C06SGFD65NC/p1744818806774219?thread_ts=1744808845.862799&cid=C06SGFD65NC
And maybe we can remove Paul, but keep Tomas.